src/main/java/com/labsynch/labseer/utils/PropertiesUtilServiceImpl.java

```diff
@@ -293,4 +298,382 @@ public void setEmailFromAddress(String emailFromAddress) {
 	public String getEmailFromAddress() {
 	    return this.emailFromAddress;
 	}
+
+
+	//CMPD REG PROPERTIES
+	String acasURL;
```

Comment:
Spacing is a little funny here, usually these have a return between them and the decorator on the next function.